helpers/util.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
-var getCurrentDocument = require("can-globals/document/document");
-var isBrowserWindow = require("can-util/js/is-browser-window/is-browser-window");
+var getCurrentDocument = require("can-dom-data-state");
```

Comment:
This isn't right.

Reply:
Wow... I have no idea how I messed that up.